vowpalwabbit/Makefile.am

```diff
@@ -4,7 +4,7 @@ liballreduce_la_SOURCES = allreduce_sockets.cc allreduce_threads.cc vw_exception
 
 bin_PROGRAMS = vw active_interactor
 
-libvw_la_SOURCES = hash.cc global_data.cc io_buf.cc parse_regressor.cc parse_primitives.cc unique_sort.cc cache.cc rand48.cc simple_label.cc multiclass.cc oaa.cc multilabel_oaa.cc boosting.cc ect.cc autolink.cc binary.cc lrq.cc cost_sensitive.cc multilabel.cc label_dictionary.cc csoaa.cc cb.cc cb_adf.cc cb_algs.cc search.cc search_meta.cc search_sequencetask.cc search_dep_parser.cc search_hooktask.cc search_multiclasstask.cc search_entityrelationtask.cc search_graph.cc parse_example.cc scorer.cc network.cc parse_args.cc accumulate.cc gd.cc learner.cc lda_core.cc gd_mf.cc mf.cc bfgs.cc noop.cc print.cc example.cc parser.cc loss_functions.cc sender.cc nn.cc bs.cc cbify.cc topk.cc stagewise_poly.cc log_multi.cc active.cc kernel_svm.cc best_constant.cc ftrl.cc svrg.cc lrqfa.cc interact.cc comp_io.cc interactions.cc vw_exception.cc allreduce_sockets.cc allreduce_threads.cc
+libvw_la_SOURCES = hash.cc global_data.cc io_buf.cc parse_regressor.cc parse_primitives.cc unique_sort.cc cache.cc rand48.cc simple_label.cc multiclass.cc oaa.cc multilabel_oaa.cc boosting.cc ect.cc autolink.cc binary.cc lrq.cc cost_sensitive.cc multilabel.cc label_dictionary.cc csoaa.cc cb.cc cb_adf.cc cb_algs.cc search.cc search_meta.cc search_sequencetask.cc search_dep_parser.cc search_hooktask.cc search_multiclasstask.cc search_entityrelationtask.cc search_graph.cc parse_example.cc scorer.cc network.cc parse_args.cc accumulate.cc gd.cc learner.cc lda_core.cc gd_mf.cc mf.cc bfgs.cc noop.cc print.cc example.cc parser.cc loss_functions.cc sender.cc nn.cc bs.cc cbify.cc topk.cc stagewise_poly.cc log_multi.cc active.cc active_cover.cc kernel_svm.cc best_constant.cc ftrl.cc svrg.cc lrqfa.cc interact.cc comp_io.cc interactions.cc vw_exception.cc allreduce_sockets.cc allreduce_threads.cc
```

Comment:
That’s part of the Model hardening & example… pull request…

From: John [mailto:notifications@github.com]
Sent: Monday, October 5, 2015 2:00 PM
To: JohnLangford/vowpal_wabbit <vowpal_wabbit@noreply.github.com>
Cc: Markus Cozowicz <marcozo@microsoft.com>
Subject: Re: [vowpal_wabbit] active cover code (#721)


In vowpalwabbit/Makefile.am<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fJohnLangford%2fvowpal_wabbit%2fpull%2f721%23discussion_r41175742&data=01%7c01%7cmarcozo%40064d.mgd.microsoft.com%7c2a27e63ee6cb4443bd1808d2cdaed30f%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=EaoxrZ7jEoxfX7YHVebw9qKLxKV6WXXTcshXaa8w4aI%3d>:

> @@ -4,7 +4,7 @@ liballreduce_la_SOURCES = allreduce_sockets.cc allreduce_threads.cc vw_exception

>

>  bin_PROGRAMS = vw active_interactor

>

> -libvw_la_SOURCES = hash.cc global_data.cc io_buf.cc parse_regressor.cc parse_primitives.cc unique_sort.cc cache.cc rand48.cc simple_label.cc multiclass.cc oaa.cc multilabel_oaa.cc boosting.cc ect.cc autolink.cc binary.cc lrq.cc cost_sensitive.cc multilabel.cc label_dictionary.cc csoaa.cc cb.cc cb_adf.cc cb_algs.cc search.cc search_meta.cc search_sequencetask.cc search_dep_parser.cc search_hooktask.cc search_multiclasstask.cc search_entityrelationtask.cc search_graph.cc parse_example.cc scorer.cc network.cc parse_args.cc accumulate.cc gd.cc learner.cc lda_core.cc gd_mf.cc mf.cc bfgs.cc noop.cc print.cc example.cc parser.cc loss_functions.cc sender.cc nn.cc bs.cc cbify.cc topk.cc stagewise_poly.cc log_multi.cc active.cc kernel_svm.cc best_constant.cc ftrl.cc svrg.cc lrqfa.cc interact.cc comp_io.cc interactions.cc vw_exception.cc allreduce_sockets.cc allreduce_threads.cc

> +libvw_la_SOURCES = hash.cc global_data.cc io_buf.cc parse_regressor.cc parse_primitives.cc unique_sort.cc cache.cc rand48.cc simple_label.cc multiclass.cc oaa.cc multilabel_oaa.cc boosting.cc ect.cc autolink.cc binary.cc lrq.cc cost_sensitive.cc multilabel.cc label_dictionary.cc csoaa.cc cb.cc cb_adf.cc cb_algs.cc search.cc search_meta.cc search_sequencetask.cc search_dep_parser.cc search_hooktask.cc search_multiclasstask.cc search_entityrelationtask.cc search_graph.cc parse_example.cc scorer.cc network.cc parse_args.cc accumulate.cc gd.cc learner.cc lda_core.cc gd_mf.cc mf.cc bfgs.cc noop.cc print.cc example.cc parser.cc loss_functions.cc sender.cc nn.cc bs.cc cbify.cc topk.cc stagewise_poly.cc log_multi.cc active.cc active_cover.cc kernel_svm.cc best_constant.cc ftrl.cc svrg.cc lrqfa.cc interact.cc comp_io.cc interactions.cc vw_exception.cc allreduce_sockets.cc allreduce_threads.cc

@eisber<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2feisber&data=01%7c01%7cmarcozo%40064d.mgd.microsoft.com%7c2a27e63ee6cb4443bd1808d2cdaed30f%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=NVrmXhrZPD7ZJI2mKPUC2Iq49%2fENlPQKEQuP84Qm35Q%3d> Why is there still an allreduce_sockets.cc here?

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fJohnLangford%2fvowpal_wabbit%2fpull%2f721%2ffiles%23r41175742&data=01%7c01%7cmarcozo%40064d.mgd.microsoft.com%7c2a27e63ee6cb4443bd1808d2cdaed30f%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=FBsNHsJX8hsA98CPmFnF5sOQbxWK6fzHyc%2fd92hT%2bj0%3d>.